client/blocks/login/style.scss

```diff
@@ -4,10 +4,15 @@
 
 .login__form-userdata {
 	label {
+		color: $gray-dark;
 		display: block;
-		line-height: 2em;
+		font-size: 14px;
```

Comment:
I'ts the same used on the signup up form. Also some of the strings are a bit long, so having it one pixel smaller helps.